android/build.gradle

```diff
@@ -33,7 +37,7 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    compile 'com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}'
     //compile 'com.squareup.okhttp3:okhttp:+'
```

Comment:
@yeomann Thanks for this commit--it got me past the first hurdle adding rn-fetch-blob to a new project. Line 40 threw an exception during build though until I wrapped the string in double quotes. Could you apply a fix like that for the single quotes in single quotes issue?

(I haven't officially been added to this project yet, but I'm working with @Traviskn to merge this request as soon as possible.)

Reply:
oh balls! my bad, it was actually suppose to be wrapped with double quotes, I have tested this multiple times, single quotes within single quotes doesn't work. let me fix it.